plottable.d.ts

```diff
@@ -2767,16 +2767,20 @@ declare module Plottable {
             getClosestPlotData(queryPoint: Point): PlotData;
             protected _isVisibleOnPlot(datum: any, pixelPoint: Point, selection: D3.Selection): boolean;
             /**
-             * Gets the bar under the given pixel position (if [xValOrExtent]
-             * and [yValOrExtent] are {number}s), under a given line (if only one
-             * of [xValOrExtent] or [yValOrExtent] are {Extent}s) or are under a
-             * 2D area (if [xValOrExtent] and [yValOrExtent] are both {Extent}s).
+             * Gets the {Plots.PlotData} that correspond to the given pixel position.
              *
-             * @param {number | Extent} xValOrExtent The pixel x position, or range of x values.
-             * @param {number | Extent} yValOrExtent The pixel y position, or range of y values.
-             * @returns {D3.Selection} The selected bar, or null if no bar was selected.
+             * @param {Point} p The provided pixel position as a {Point}
+             * @return {Plots.PlotData} The plot data that corresponds to the {Point}.
              */
-            getBars(xValOrExtent: number | Extent, yValOrExtent: number | Extent): D3.Selection;
+            plotDataAt(p: Point): PlotData;
+            /**
+             * Gets the {Plots.PlotData} that correspond to a given xRange/yRange
+             *
+             * @param {Extent} xRange The specified range of x values
+             * @param {Extent} yRange The specified range of y values
+             * @return {Plots.PlotData} The plot data that corresponds to the ranges
+             */
+            plotDataIn(xRange: Extent, yRange: Extent): PlotData;
```

Comment:
I vaguely recall an in-person discussion about this with @jtlan where we came to the conclusion that they should be named `xRange` and `yRange` because "range" is more or less the only part of an `Extent` that we care about. I could be wrong though..